src/main/resources/parsingPassword.txt

```diff
@@ -0,0 +1 @@
+test
```

Comment:
when i put it in the global config, i can't put it in gitignore, bc i shouldn't put the global config in gitignore